cloudhub-2/modules/ROOT/pages/ch2-patch-updates.adoc

```diff
@@ -198,6 +198,15 @@ include::anypoint-cli::partial$mule-4.5-release-channels.adoc[]
 When an automatic update occurs, the audit log for your application adds an entry by user Anypoint Staff, which shows the date and time an update occurred and whether it was successful.
 If the update is successful, no action is required. If your application does not support zero-downtime updates or requires a manual update, you can manually apply the date patch released in the first full week of the month in week 1 or week 2. See xref:ch2-patch-updates.adoc#update-the-runtime-version-manually[Update the Runtime Version Manually].
 
+== Check Your Application Status After an Automated Update
+
+After a new automated update occurs, you can check your xref:managing-deployed-applications.adoc#application_status_states[applications status state] in the *Status* column in Runtime Manager's *Applications* tab.
+
+You can also check your application logs to verify if your application was deployed correctly and take any remediation actions in the event of some auto-minor update-related issue. For more information, visit xref:ch2-view-logs.adoc#view-logs[View Logs].
+
+To be notified by e-mail when an error occurs after an automated update, you can xref:ch2-config-app-alerts.adoc[create an alert]. 
```

Comment:
E-mail alerts for auto update failures is not yet possible in CloudHub 2.0, lets remove this